jerry-port/default/default-module.c

```diff
@@ -150,7 +150,15 @@ jerry_port_normalize_path (const char *in_path_p,   /**< input file path */
 
   if (norm_p != NULL)
   {
+    size_t i;
     ret = strnlen (norm_p, out_buf_size);
+    for (i = 0; i < ret; i += 1)
+    {
+      if (norm_p[i] == '\\')
+      {
+        norm_p[i] = '/';
+      }
+    }
```

Comment:
Why is this necessary? Back slash is still the primary directory separator on Windows. Forward slashes (the alternative direcory separator) are canonicalized to back slashes. So why do double work?

Reply:
For fixes tests on win32. Hard to fix

Reply:
Sorry, but this is no explanation. Please, provide example where this is necessary.

Reply:
```tests\jerry\es.next\module-resource-name.js
import { getName, getNamePromise } from "./module-resource-name-export.mjs"

assert(getName().endsWith("/module-resource-name-export.mjs") === true);

var collector = {};
getNamePromise(collector).then(() => { collector["end"] = resourceName(); });

function __checkAsync() {
  assert(collector["start"].endsWith("/module-resource-name-export.mjs") === true);
  assert(collector["middle"].endsWith("/module-resource-name-export.mjs") === true);
  assert(collector["end"].endsWith("/module-resource-name.js") === true);
  assert(Object.keys(collector).length === 3);
}
```
This testcase want endes with "/.
That's unix style.

Reply:
But this is more of an error (invalid assumption) in the test case, not in the port module implementation.

Reply:
> But this is more of an error (invalid assumption) in the test case, not in the port module implementation.

OK, I'll modify the test instead